app/assets/stylesheets/elements/_entry.scss

```diff
@@ -16,11 +16,11 @@ article {
     text-transform: uppercase;
 
     a {
-      color: $base-font-color;
+      color: red;
```

Comment:
Color `red` should be written in hexadecimal form as `#ff0000`<br>Color literals like `red` should only be used in variable declarations; they should be referred to via variable everywhere else.